.github/PULL_REQUEST-TEMPLATE.md

```diff
@@ -20,10 +16,14 @@
 * [ ] I have added/updated functional tests where appropriate
 * [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)
 
-__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__
+__An exhaustive list of peer review checks can be
```

Comment:
Did you intend to push the changes to this template? Nothing to do with the linked ticket...

Reply:
this was done at the pre-commit
all the files recovered during the merge have been reformatted